cowboys/routers_cowboys.py

```diff
@@ -0,0 +1,29 @@
+from fastapi import APIRouter, Depends, Body, status
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Session
+
+from cowboys import crud, schemas
+from settings import bd_coneccion
+
+router = APIRouter()
+
+@router.post ("/Cowboys", response_model = schemas.CrearCowboyRespuesta)
```

Comment:
```suggestion
@router.post ("/cowboys", response_model = schemas.CrearCowboyRespuesta)
```

Habría que poner la palabra `cowboys` con minusculas.